src/cpr_data_access/models/search.py

```diff
@@ -81,10 +81,10 @@ def keyword_filters_must_be_valid(cls, keyword_filters):
         """Validate that the keyword filters are valid."""
         if keyword_filters is not None:
             for field_key, values in keyword_filters.items():
-                if field_key not in filter_fields:
+                if field_key not in filter_fields.values():
```

Comment:
Does filter fields still need to be a dictionary? I can't see it being used anywhere else and we only ever want the values?

Reply:
It's required in the backend to map the frontend keys to the vespa keys.

Reply:
Ah good shout